cmd/subnetcmd/deploy.go

```diff
@@ -221,19 +228,27 @@ func deploySubnet(cmd *cobra.Command, args []string) error {
 		}
 		return nil
 
-	case models.Fuji: // just make the switch pass
-		if keyName == "" {
-			keyName, err = captureKeyName()
+	case models.Fuji:
+		if !useLedger && keyName == "" {
+			var err error
+			useLedger, err = app.Prompt.CaptureYesNo("Use ledger as private key to issue the transaction?")
```

Comment:
I would switch to a list selection here, you may want to make that a shared method in `prompts` something like `ChooseKeyOrLedger` since we will need to use it a few places.

Reply:
It would be fine to return a bool with that method.